requirements.txt

```diff
@@ -1,4 +1,3 @@
-Django==1.6.1
```

Comment:
End of an era.  The precursor to Uber used at MAGFest 4 was entirely a Django app.  Now the last remnants of the Old Republic have been swept away forever.